index.html

```diff
@@ -21,14 +21,16 @@
         <select id="sort-options">
           <option value="title">Sort by title</option>
           <option value="movie_year">Sort by year</option>
-          <option value="rating">Sort by rating</option>
+          <option value="rating">Sort by rating </option>
         </select>
+
       </span>
 
         <nav>
           <li>
             <input type="search" id="keyword-search" placeholder="search" required />
             <i id="search-icon" class="fa-solid fa-magnifying-glass"></i>
+            <input type="number" id="year-filter" placeholder="Filter by year..." style="color: aliceblue;" />
```

Comment:
```suggestion
            <input type="number" id="year-filter" placeholder="Filter by year..." style="color: aliceblue;" min="0" />
```
It's currently possible to filter by negative years. I don't think they had movies at this time.